oauthproxy.go

```diff
@@ -858,9 +858,17 @@ func (p *OAuthProxy) Proxy(rw http.ResponseWriter, req *http.Request) {
 	session, err := p.getAuthenticatedSession(rw, req)
 	switch err {
 	case nil:
+		//<!--COCO Begin-->
+		// COCO: check session token present in cookie or not
+		cookie, err := req.Cookie("sessionToken")
+		if err != nil {
+			fmt.Printf("can't find sessionToken redireacting to logout page")
```

Comment:
- Why are we using fmt.Printf(), when there are no variables to be passed. Is this the default Print statement?

Reply:
no, we can use logger.printf i think its better to use logger.Printf() many other places they use as logger message